src/bluesky/consolidators.py

```diff
@@ -111,6 +112,18 @@ def __init__(self, stream_resource: StreamResource, descriptor: EventDescriptor)
         self._has_skips: bool = False
         self._seqnums_to_indices_map: Dict[int, int] = {}
 
+        # Set the maximal chunk size; if not specified -- assume single chunk for the entire file
+        self.chunk_size: Sequence[int] = self._sres_parameters.get("chunk_size", self.shape)
+        if isinstance(self.chunk_size, int):
+            # Only the left-most dimension (rows) is chunked; other dimensions are in a single chunk
+            self.chunk_size = (self.chunk_size, *self.datum_shape)
+        else:
+            # Chunk size is specified for each dimension separately as a list or a tuple
+            if len(self.chunk_size) == len(self.datum_shape):
+                self.chunk_size = (0, self.chunk_size)  # Assume the leading dimension is not chunked
+            elif len(self.chunk_size) != len(self.shape):
+                raise RuntimeError(f"Dimensions of chunks {self.chunk_size} do not match the data {self.shape}.")
+
```

Comment:
Didn't we settle on `chunk_shape` being in the sres parameters? Rather than `chunk_size`?

Reply:
Yes, we did!
Sorry, I haven't realized you were getting notifications and paying attention to this PR. I was just updating my working branch with the current main (not even sure why I pushed it...). I will rebase it again once your `chunk_shape` PR is merged into `main`.